synapse/third_party_rules/access_rules.py

```diff
@@ -167,8 +171,44 @@ def on_create_room(self, requester, config, is_requester_admin) -> bool:
                 if not allowed:
                     raise SynapseError(400, "Invalid power levels content")
 
+                use_default_power_levels = False
+
+        # If power levels were not overridden by the user, override with DINUM's preferred
+        # defaults instead
+        if use_default_power_levels:
+            config["power_level_content_override"] = self._get_default_power_levels(
+                requester
+            )
```

Comment:
it might be worth calling out in the docstring that we modify the `power_level_content_override` (as well as "append an event with the default rule").